Phar/Phar.php

```diff
@@ -109,7 +112,8 @@ public function addFile(
     #[TentativeType]
     public function addFromString(
         #[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $localName,
-        #[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $contents
+        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $contents = '',
```

Comment:
Looks like `$contents = '',` doesn't actually have a default value: https://3v4l.org/fb4Fq

Reply:
Yeah, according to reflection this parameter doesn't have a default value, but still, it's optional and unfortunately, in terms of stubs, we don't have any other mechanism to say that parameter is optional other than adding default value or `[optional]` to PhpDoc. The latter wouldn't work since it would be applied for all PHP versions, so the default value is the only way here.